obspy/io/seiscomp/sc3ml.py

```diff
@@ -398,11 +401,11 @@ def _read_channel(inventory_root, cha_element, _ns):
     if sensor_element is not None:
         response_id = sensor_element.get("response")
         if response_id is not None:
-            resp_type = response_id.split("#")[0]
-            if resp_type == 'ResponsePAZ':
-                search = "responsePAZ[@publicID='" + response_id + "']"
-                response_element = inventory_root.find(_ns(search))
-            elif resp_type == 'ResponsePolynomial':
+            resp_type = 'ResponsePAZ'
+            search = "responsePAZ[@publicID='" + response_id + "']"
+            response_element = inventory_root.find(_ns(search))
+            if response_element is None:
+                resp_type == 'ResponsePolynomial'
```

Comment:
There's also `ResponseFIR` tags.. they should also be searched, no?

I'd also propose to sanitize the lookup a bit so that problems with one publicID being multiplied in the file are caught:

```python
            response_elements = []
            for resp_type in ['responsePAZ', 'responsePolynomial', 'responseFIR']:
                search = "{}[@publicID='{}']".format(resp_type, response_id)
                response_elements += inventory_root.findall(_ns(search))
            if len(response_elements) == 0:
                msg = ("Could not find response tag with public ID "
                       "'{}'.".format(response_id))
                raise ObsPyException(msg)
            elif len(response_elements) > 1:
                msg = ("Found multiple matching response tags with the same "
                       "public ID '{}'.".format(response_id))
                raise ObsPyException(msg)
            response_element = response_elements[0]
```